packages/common/src/types.ts

```diff
@@ -48,6 +48,7 @@ export interface HardforkConfig {
   name: Hardfork | string
   block: number | null // null is used for hardforks that should not be applied -- since `undefined` isn't a valid value in JSON
   ttd?: bigint | string
+  timestamp?: number | string
```

Comment:
Is it possible for this to be `null` if a timestamp hardfork is read in from a geth genesis config file?

Reply:
haven't seen null specified in geth genesis, but i can make sure we remove those hardforks from consideration where their block/time is specified as null